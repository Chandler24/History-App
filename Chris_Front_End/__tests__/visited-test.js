import 'react-native';
import React from 'react';
import Visited from '../src/pages/visited';
import renderer from 'react-test-renderer';

it ('renders correctly', () => {
  const snap = renderer.create(
    <Visited />
  ).toJSON();
  expect(snap).toMatchSnapshot();
});