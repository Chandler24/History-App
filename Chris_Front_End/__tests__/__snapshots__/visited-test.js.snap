// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#2e88ff",
      "flexGrow": 1,
    }
  }
>
  <View>
    <RCTScrollView
      horizontal={false}
      showsVerticalScrollIndicator={false}
    >
      <View>
        <View
          style={
            Object {
              "margin": 6,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "#005ccb",
              "borderColor": "#005ccb",
              "borderRadius": 25,
              "borderWidth": 2,
              "height": 130,
              "marginBottom": 10,
            }
          }
        >
          <Image
            source={1}
            style={
              Object {
                "borderRadius": 20,
                "flex": 1,
                "height": null,
                "resizeMode": "cover",
                "width": null,
              }
            }
          />
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "#ffffff",
                "fontSize": 20,
                "fontWeight": "500",
                "textAlign": "center",
              }
            }
          >
            Orlando City Hall
             
          </Text>
        </View>
      </View>
    </RCTScrollView>
  </View>
</View>
`;
